bulk_site_creator/templates/bulk_site_creator/new_job.html

```diff
@@ -157,6 +156,12 @@ <h5 class="modal-title" id="staticBackdropLabel">Confirm Canvas Course Site Crea
                 </div>
               </div>
             </div>
+            <!-- If no potential_course_sites -->
+            {% else %}
+                <hr>
+                <div class="row">
+                    <h4>No courses ready for Canvas course site creation</h4>
```

Comment:
Maybe conditionally display this block so it doesn't display on the initial load of the page. You could do a check something along the lines of {% if selected_term_id %}, because that context value will only be provided after at least one search has been conducted.
Otherwise it could appear that there are no courses available for the term + dept/coursegroup that is displayed on the initial load of the page

Reply:
Good idea, I'll do that!